.github/workflows/build-test.yml

```
@@ -55,7 +55,7 @@ jobs:
     needs:
       - prepare-environment
       - build-development-image
-    uses: open-space-collective/open-space-toolkit/.github/workflows/test.yml@main
+    uses: open-space-collective/open-space-toolkit/.github/workflows/test.yml@build/update-github-actions-extensions
```

```suggestion
    uses: open-space-collective/open-space-toolkit/.github/workflows/test.yml@main
```
Apply once base MR is merged